### DISCUSSION TITLE
Mods in server's playlist

### DISCUSSION THREAD

**Discussion by MRcreeper176** [efroemling/ballistica#664] [Q&A] [answered]
What should i do if i want to add some maps, minigames, etc. to my server's playlist? I tried this but i just receive error for every minigame i added to my playlist. What should i do?
![20240224_192723](https://github.com/efroemling/ballistica/assets/156354493/b493d63b-bbce-41a4-a9c6-31b762171372)

**Comment by Loup-Garou911XD** [comment, marked as answer]
You need to add the minigame file to mods directory

**Reply by MRcreeper176** [reply]
Can you give me the path of mods directory please?

**Reply by Loup-Garou911XD** [reply]
dist/ba_root/mods/

**Reply by MRcreeper176** [reply]
Thanks👍👍👍
